docs/modules/position.md

```diff
@@ -0,0 +1,609 @@
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to store the score to.
+
+    (execution) `at <entity>` or `rotated as <entity>`
+    : The rotation to get.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (score) `@s bs.rot.v`
+    : The vertical rotation of the position.
+:::
+::::
+
+> **Credits**: Aksiome, Leirof, theogiraudet
+
+---
+
+### Normalize rotation
+
+**`#bs.position:normalize_rot`**
+
+Allows to normalize the oriantations (replace the `bs.rot.h` and `bs.rot.v` scores respectively in the interval [0;360[ and [0;180[)
+
+> **Credits**: Leirof
+
+---
+
+### Get relative
+
+**`#bs.position:get_relative_ata {scale:<scaling>}`**
+
+Obtain the execution position of the function relatively to the position of the executing entity.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity, considered at the reference point.
+
+    (execution) `at <entity>` or `positioned <x> <y> <z>`
+    : The position te get the relative position from.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (score) `$position.get_relative_ata.[x,y,z] bs.out`
+    : The relative coordinates.
+
+Example
+
+:   Get your position relative to the nearest Creeper (without scaling):
+
+    ```
+    # Once
+    execute as @s at @e[type=creeper,limit=1,sort=nearest] run function #bs.position:get_relative_ata {scale:1}
+
+    # See the result
+    tellraw @a [{"text": "Relative position : X=", "color": "dark_gray"},{"score":{"name":"$position.get_relative_ata.x", "objective": "bs.out"}, "color": "gold"},{"text":", Y=", "color": "dark_gray"},{"score":{"name":"$position.get_relative_ata.y", "objective": "bs.out"},"color":"gold"},{"text":", Z=","color":"dark_gray"},{"score":{"name":"$position.get_relative_ata.z","objective":"bs.out"},"color":"gold"}]
+    ```
+
+> **Credits**: Aksiome, Leirof
+
+---
+
+### Get distance
+
+::::{tab-set}
+:::{tab-item} Normal
+
+**`#bs.position:get_distance_ata {scale:<scaling>}`**
+
+Calculates the distance between the source entity and the execution position of the function.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity positioned at the reference point.
+
+    (execution) `at <entity>` or `positioned <x> <y> <z>`
+    : The position to get the distance from.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (score) `$position.get_distance_ata bs.out`
+    : The distance between the two positions.
+
+Example
+
+:   Calculate the distance between you and the nearest sheep (in blocks, without scaling):
+
+    ```
+    # Once
+    execute at @e[type=sheep,limit=1,sort=nearest] run function #bs.position:get_distance_ata {scale:1}
+
+    # See the result
+    tellraw @a [{"text": "Distance: ", "color": "dark_gray"},{"score":{"name":"$position.get_distance_ata", "objective": "bs.out"}, "color": "gold"}]
+    ```
```

Comment:
Always output 1

Reply:
MOJANNNNNG, they just changed how numbers are formatted when using a macro so the float suffix was missing...